.autoupdate/preupdate

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This script is called by our weekly dependency update job in Jenkins
+
+pip3 install uv > rialto-airflow.txt &&
+    ~/.local/bin/uv lock --upgrade >> rialto-airflow.txt
```

Comment:
add a third line like `uv pip compile pyproject.toml -o requirements.txt --upgrade` from readme so that we also generate an updated `requirements.txt`